### PR TITLE
[Event Hubs] Update HISTORY, README and version bump

### PR DESCRIPTION
**Pull request by YijunXieMS**

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
 # Release History
 
-## 5.0.2 (Unreleased)
-
+## 5.1.0b1 (Unreleased)
```
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhub/CHANGELOG.md:3]: do we want to update release date now or in a separate PR

File: sdk/eventhub/azure-eventhub/README.md [hunk 2]
```diff
@@ -115,8 +114,10 @@ The following sections provide several code snippets covering some of the most c
 - [Inspect an Event Hub](#inspect-an-event-hub)
 - [Publish events to an Event Hub](#publish-events-to-an-event-hub)
 - [Consume events from an Event Hub](#consume-events-from-an-event-hub)
+- [Consume events from an Event Hub in Batches](#consume-events-from-an-event-hub-in-batches)
```
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhub/README.md:117]: lower case "in batch" to align with the title of asynchronous one

File: sdk/eventhub/azure-eventhub/README.md [hunk 3]
```diff
@@ -187,9 +185,35 @@ with client:
...
+    logger.info("Received event from partition {}".format(partition_context.partition_id))
+    partition_context.update_checkpoint()
+
+with client:
+    client.receive_batch(
+        on_event_batch=on_event_batch, 
+        starting_position="-1",  # "-1" is from the beginning of the partition.
+    )
+    # receive events from specified partition:
+    # client.receive(on_event=on_event, partition_id='0')
```
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhub/README.md:212]: `receive_batch` here

File: sdk/eventhub/azure-eventhub/README.md [hunk 4]
```diff
@@ -257,6 +279,39 @@ if __name__ == '__main__':
...
+
+async def receive_batch():
+    client = EventHubConsumerClient.from_connection_string(connection_str, consumer_group, eventhub_name=eventhub_name)
+    async with client:
+        await client.receive_batch(
+            on_event_batch=on_event_batch,
+            starting_position="-1",  # "-1" is from the beginning of the partition.
+        )
+        # receive events from specified partition:
+        # await client.receive(on_event=on_event, partition_id='0')
```
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhub/README.md:308]: same comment as the sync example

File: sdk/eventhub/azure-eventhub/CHANGELOG.md (hunk 1 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhub/CHANGELOG.md]: Changed to 2020-04-06

File: sdk/eventhub/azure-eventhub/README.md (hunk 2 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhub/README.md]: fixed

File: sdk/eventhub/azure-eventhub/README.md (hunk 3 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhub/README.md]: corrected comments

File: sdk/eventhub/azure-eventhub/README.md (hunk 4 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhub/README.md]: corrected

**Pull request merged**